components/MemberComponent.vue

```diff
@@ -1,9 +1,13 @@
 <template>
     <div class="member flex flex-col items-center text-center">
-        <img
-            data-testid="member-avatar"
-            :src="avatarImg"
-            class="member-image object-cover rounded-full w-32 h-32"
+        <SmoothLoadingImage
+            :imgSrc="avatarImg"
```

Comment:
This part is PERFECT linting/styling, if you can do this for your SmoothLoadingImage component as well would be great ✨